src/test/rustdoc-gui/headings.goml

```diff
@@ -154,3 +154,15 @@ assert-css: ("h2#top-doc-prose-title", {"font-size": "20.8px"})
 assert-css: ("h2#top-doc-prose-title", {"border-bottom-width": "1px"})
 assert-css: ("h3#top-doc-prose-sub-heading", {"font-size": "18.4px"})
 assert-css: ("h3#top-doc-prose-sub-heading", {"border-bottom-width": "1px"})
+
+goto: file://|DOC_PATH|/staged_api/struct.Foo.html
+show-text: true
+assert-css: (".since", {"color": "rgb(128, 128, 128)"})
```

Comment:
Just in case, can you also set the theme for the first to light too please? (In case in a potential future we change the default theme...)